style.css

```
@@ -0,0 +1,334 @@
+  font-family: inherit;
+  font-weight: 600;
+  background-color: hsl(189, 41%, 97%);
+  color: hsl(183, 100%, 15%);
+  border-radius: 4px;
+  transition: all .1s;
+}
+
+.main-input:focus {
+  border: 2px solid hsl(172, 67%, 45%);
+}
+
+
+
+.input-title-box {
+  display: flex;
+  justify-content: space-between;
+}
+
+.bill-error,
+.people-error,
+.tip-error {
+  color: red;
+}
+
+/*disabled the arrows in the number inputs*/
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+.input-box i {
+  position: absolute;
+  top: 60%;
+  left: 6%;
+}
+
+
+/*Percents box*/
+.btn-box {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 12px;
+}
+
+.btn-percent {
+  border: none;
+  height: 52px;
+  background-color: hsl(183, 100%, 15%);
+  color: #fff;
+  font-family: inherit;
+  font-size: 22px;
+  border-radius: 4px;
+  font-weight: 600;
+  cursor: pointer;
+  transition: all .4s;
+}
+
+.btn-percent:hover {
+  background-color: hsl(184, 79%, 79%);
+  color: hsl(183, 100%, 15%);
+}
+
+.custom-input {
+  font-size: 24px;
+  background-color: hsl(189, 41%, 97%);
+  border: none;
+  font-weight: 600;
+  font-family: inherit;
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+  padding: 0 12px;
+  outline: none;
+  text-align: right;
+  color: hsl(183, 100%, 15%);
+  border-radius: 4px;
+
+}
+
+.custom-input:focus {
+  border: 2px solid hsl(172, 67%, 45%);
+}
+
+.not-valid:focus {
+  border: 2px solid red;
+}
+
+.custom-input::placeholder {
+  color: hsl(186, 14%, 43%);
+  text-align: center;
+
+}
+
+.clicked {
+  color: hsl(183, 100%, 15%);
+  background-color: hsl(172, 80%, 51%);
+}
+
+.clicked:hover {
+  color: hsl(183, 100%, 15%);
+  background-color: hsl(172, 80%, 51%);
+}
+
+/* Dashboard box */
+.dashboard-box {
+  color: #fff;
+  background-color: hsl(183, 100%, 15%);
+  width: 90%;
+  border-radius: 8px;
+  padding: 48px 36px;
+
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+}
+
+.result-box {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.tip-amount,
+.total-amount {
+  font-size: 42px;
+  color: hsl(172, 80%, 51%);
+}
+
+.title-box {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+}
+
+.title-box h4 {
+  font-size: 14px;
+  color: hsl(189, 41%, 97%)
+}
+
+.title-box p {
+  font-size: 12px;
+  color: hsl(184, 14%, 56%);
+  font-weight: 600;
+}
+
+.reset-btn {
+  border: none;
+  padding: 12px 24px;
+  border-radius: 4px;
+  color: hsl(183, 100%, 15%);
+  background-color: hsl(186, 14%, 43%);
+  font-family: inherit;
+  font-weight: 600;
+  font-size: 18px;
+  margin-top: auto;
+  transition: all .3s;
+}
+
+
+.on {
+  color: hsl(183, 100%, 15%);
+  background-color: hsl(172, 80%, 51%);
+  cursor: pointer;
+
+}
+
+.on:hover {
+  background-color: hsl(184, 79%, 79%);
+}
+
+
+/* Media query */
+@media(max-width:375px) {
+  .container {
+    padding-top: 50px;
+    justify-content: flex-start;
```

Try not to use padding-top in px, using flex and grid we can apply gap and control the element position without needing to add padding

---

The gap is between 2 elements, I don't have any other element to make gap between the container to that element, the only thing I thought to do is to add ".container::before" and then it "creates a new element" above the container and makes some space

```suggestion
    .container::before{
        content: "";
     }
```